projects/packages/videopress/src/client/state/constants.js

```diff
@@ -24,6 +24,11 @@ export const REMOVE_VIDEO = 'REMOVE_VIDEO';
 export const SET_IS_FETCHING_UPLOADED_VIDEO_COUNT = 'SET_IS_FETCHING_UPLOADED_VIDEO_COUNT';
 export const SET_UPLOADED_VIDEO_COUNT = 'SET_UPLOADED_VIDEO_COUNT';
 
+/*
+ * Accepted file extensions
+ */
+export const VIDEO_EXTENSIONS = [ 'ogv', 'mp4', 'mov', 'wmv', 'avi', 'mpg', '3gp', '3g2', 'm4v' ];
```

Comment:
Don't think this is the place to put this value. IMO it should exists inside the `video-upload-area`component

Reply:
The site should define these values and not be hardcoded. Maybe we can create a follow-up PR to exposes the mime types to the client.

Reply:
created an issue https://github.com/Automattic/jetpack/issues/26468

Reply:
This should not be the place to add this mime for now